Mathlib/Data/MvPolynomial/Basic.lean

```diff
@@ -730,6 +730,13 @@ theorem coeff_X_mul (m) (s : σ) (p : MvPolynomial σ R) :
   (coeff_monomial_mul _ _ _ _).trans (one_mul _)
 #align mv_polynomial.coeff_X_mul MvPolynomial.coeff_X_mul
 
+/-- MvPolynomials in an empty type of variables are always constant -/
+lemma eq_C_of_empty [IsEmpty σ] (p : MvPolynomial σ R) :
```

Comment:
I think this should be proved via `MvPolynomial.C_surjective`. How about placing the following just below `coeff_C`:
```lean
lemma eq_C_of_isEmpty [IsEmpty σ] (p : MvPolynomial σ R) :
    p = C (p.coeff 0) := by
  obtain ⟨x, rfl⟩ := C_surjective σ p
  simp
```